package.json

```diff
@@ -11,6 +11,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
+    "codecov.io": "0.1.6",
```

Comment:
Definitely yalls call on that. I always thought that for 0.x versions, semver would not actually satisfy versions other than the one specified